joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50
# Controller to control robot base joints
left_wheel_velocity_controller: 
  type: "velocity_controllers/JointVelocityController"
  joint: "left_wheel_joint"
  pid: {p: 10.0, i: 0.1, d: 10.0}

right_wheel_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "right_wheel_joint"
  pid: {p: 10.0, i: 0.1, d: 10.0}

font_wheel_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "font_wheel_joint"
  pid: {p: 10.0, i: 0.1, d: 10.0}

# Controller to control robot arm joints
arm_1_joint_controller:
  type: position_controllers/JointPositionController
  joint: arm1_joint

arm_2_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: arm2_joint